### ISSUE TITLE
Prompted to restart to update multiple times for every monthly version

### ISSUE THREAD

**Issue by jasonlewicki** [microsoft/vscode#107530]
- VSCode Version:  1.49.2
- OS Version: macOS Catalina 10.15.7

Steps to Reproduce:

1. Seems like just having vscode open will eventually prompt for an update of the same version

Does this issue occur when all extensions are disabled?: Yes

Every few days the gear has a new notification, saying that I should restart to update vscode. More often than not it's the same version that was already updated several times prior (unless an actual new update has been released). The release page for the month shows as well.

**Comment by kieferrm** [comment]
@jasonlewicki to clarify. What you see is (as an example):
- You have version 1.49.1 installed
- You restart VS Code to update to 1.49.2
- After a while you see the badge on the gear again to update.
- You restart VS Code, you still sit on 1.49.2 (because that's the latest stable version of VS Code).

**Comment by jasonlewicki** [comment]
@jasonlewicki ahh, ok I think i see what's going on. The monthly updates have minor updates. What I'm seeing on my end is:

"Hey, restart to update"
*restarts*
*tab popup that says: "Welcome to the AUGUST 2020 update! and in very small print is the incremental update version"

then a few days later:
"Hey, restart to update"
*restarts*
*tab popup that says: "Welcome to the AUGUST 2020 update! and in very small print is the incremental update version"

I guess, for my perspective, to make it not feel like groundhog day, it would be great to have the release notes say so, and/or when this update was released (and in larger font). Because right now it just looks like the same August update over and over..

Thanks

**Comment by kieferrm** [comment]
For reference, this is what a minor version update looks like:

![image](https://user-images.githubusercontent.com/4674940/94499273-151d1a00-01b1-11eb-9d3e-1bad40e9c32c.png)

**Comment by kieferrm** [comment]
@gregvanl do you remember why we chose to not highlight the incremental version number more?

**Comment by jasonlewicki** [comment]
The highlight might depend on if the client had already installed a previous major/minor version.

**Comment by gregvanl** [comment]
@kieferrm I know we decided to leave the version in the main header as major.minor and I don't think users would notice if we changed the point release value.
I think the current bold **Update X.YY.Z** in a separate section below the main header is quite visible and is a good balance between alerting users and letting them ignore point release details.
We could increase the font size by making these H2s or adding update-specific CSS but right now we're just using standard Markdown paragraphs with bold text.
